aspnetcore/fundamentals/app-state.md

```diff
@@ -68,35 +67,32 @@ Choosing a TempData provider involves several considerations, such as:
 
 <a name="config-temp"></a>
 ### Configure the TempData provider
+
 # [ASP.NET Core 2.x](#tab/aspnetcore2x)
 
-In ASP.NET Core 2.0 and later, the cookie-based TempData provider is enabled by default. Use the following code to change the TempData provider.
+If targeting .NET Framework, add the [Microsoft.AspNetCore.Session](https://www.nuget.org/packages/Microsoft.AspNetCore.Session) NuGet package to your project.
```

Comment:
Do we call this out elsewhere to say that you don't need this package reference if you're referencing `Microsoft.AspNetCore.All`? Plus, maybe it needs to come after the next line to say here are the things you need to do to configure TempData to use a session based provider.

Reply:
We've been making it a point to call that out as we make 2.0 updates to docs.